packages/css/src/components/hint/hint.scss

```diff
@@ -0,0 +1,9 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+.ams-hint {
+  color: var(--ams-hint-color);
+  white-space: nowrap;
```

Comment:
This is a little risky for text that can be input by users. Wouldn't `display: inline-block;` achieve the same goal and allow the hint text to wrap when necessary?

Reply:
Yeah, that was my solution before (`display: inline-flex` actually), but @VincentSmedinga didn't agree. He suggested `white-space: nowrap`. I've also looked at `text-wrap: nowrap`.

Reply:
I will change it back to `flex: inline-block`.

Reply:
Done!